examples/no-code/package.json

```diff
@@ -8,12 +8,12 @@
         "serve": "vite preview"
     },
     "dependencies": {
-        "@mui/material": "^5.15.20",
+        "@mui/material": "^5.16.12",
         "ra-data-local-storage": "^5.0.0",
         "ra-no-code": "^5.0.0",
-        "react": "^18.3.1",
+        "react": "18.3.1",
```

Comment:
same
(won't keep repeating it, appears multiple times across the changed files)